plugins/modules/azure_rm_adapplication.py

```diff
@@ -387,6 +454,14 @@
         type='str'
     )
 )
+
+optional_claims_spec = dict(
+    access_token=dict(type='list', elements='dict', options=claims_spec),
+    id_token=dict(type='list', elements='dict', options=claims_spec),
+    saml2_token=dict(type='list', elements='dict', options=claims_spec),
+    type=dict
```

Comment:
What's this?
```suggestion

```

Reply:
This input should be a dict, I'll condense it into the arg spec to make it clearer